cql/src/main/java/org/apache/cassandra/stargate/transport/internal/SchemaAgreement.java

```diff
@@ -24,14 +24,14 @@
   private static final ScheduledThreadPoolExecutor EXECUTOR = new ScheduledThreadPoolExecutor(1);
 
   public static CompletableFuture<? extends Result> maybeWaitForAgreement(
```

Comment:
Technically this would be API change; should there be a deprecated variant with old signature, calling new one with `persistence.connection()`? I don't know if anyone/anything outside Stargate calls this method, if not no need to bother I guess.

In future we probably need to consider API changes more carefully and add javadoc tags ("@since [version]").

Reply:
`maybeWaitForAgreement()` is internally implementation only.